db/migrate/20230124144934_create_posts.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+
+      t.timestamps
```

Comment:
- [ ] Kindly create the foreign key between users table and posts table as `author_id`.
- [ ] Please add `title`, `text`, `comments_counter`, and `likes_counter` columns as mentioned in the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
I just updated that `t.references` line with the `author_id`, I also forgot to push my changes in that table, it's done now, thank you.